src/rgb.rs

```diff
@@ -127,21 +202,25 @@ impl<T: Float> Alpha<Rgb<T>, T> {
     }
 }
 
-impl<T: Float> FromColor<T> for Rgb<T> {
-    fn from_xyz(xyz: Xyz<T>) -> Self {
+impl<Wp, T> FromColor<Wp, T> for Rgb<Wp, T>
+    where T: Float,
+        Wp: WhitePoint<T>
+{
+    fn from_xyz(xyz: Xyz<Wp, T>) -> Self {
         Rgb {
             red: xyz.x * flt(3.2404542) + xyz.y * flt(-1.5371385) + xyz.z * flt(-0.4985314),
             green: xyz.x * flt(-0.9692660) + xyz.y * flt(1.8760108) + xyz.z * flt(0.0415560),
             blue: xyz.x * flt(0.0556434) + xyz.y * flt(-0.2040259) + xyz.z * flt(1.0572252),
+            white_point: PhantomData,
```

Comment:
This is only for Srgb D65, but without adding Rgb variants not much can be done here. This can be taken care of while adding the Rgb variants which I will implement next.